doc/build.rst

```diff
@@ -70,34 +87,44 @@ Install cmake >= v3.11. E.g., the following commands install cmake v3.15.6.
    wget https://github.com/Kitware/CMake/releases/download/v3.15.6/cmake-3.15.6-Linux-x86_64.sh
    sudo bash cmake-3.15.6-Linux-x86_64.sh --skip-license --prefix=/usr/local
 
-***************************
-Building the Shared Library
-***************************
+***********************
+Building Secure XGBoost
+***********************
 
-Our goal is to build the shared library:
+Our goal is to build the shared library, along with the enclave:
 
 - On Linux the target library is ``libxgboost.so``
+- The target enclave is ``xgboost_enclave.signed``
 
 The minimal building requirement is
 
 - A recent C++ compiler supporting C++11 (g++-4.8 or higher)
 - CMake 3.11 or higher
 
-Building on Ubuntu
+Building the Targets
 ==================
 
-On Ubuntu, one builds XGBoost by running CMake:
+1. Clone the repository recursively, and check out the required branch:
```

Comment:
Maybe we can remove "and check out the required branch." I think this is a holdover from the hackathon branch?

Reply:
Fixed!